src/abstractdataframe/abstractdataframe.jl

```diff
@@ -565,251 +565,6 @@ $METADATA_FIXED
 @inline Base.last(df::AbstractDataFrame, n::Integer; view::Bool=false) =
     view ? Base.view(df, max(1, nrow(df)-n+1):nrow(df), :) : df[max(1, nrow(df)-n+1):nrow(df), :]
 
-
```

Comment:
Why not move other things out of this file? It's kind of weird to have files with a well-defined scope, and this file with heterogeneous stuff.

Reply:
This file, historically, holds all methods that take `AbstractDataFrame`. That is why it is heterogeneous. I have moved out parts that were large enough (several hundred lines). Other functions are mostly small ones and did not seem as a group of similar functions that could be grouped together.